packages/typespec-test/test/batch_modular/generated/typespec-ts/test/public/sampleTest.spec.ts

```diff
@@ -1,20 +1,18 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { Recorder } from "@azure-tools/test-recorder";
-import { assert } from "chai";
 import { createRecorder } from "./utils/recordedClient.js";
-import { Context } from "mocha";
+import { assert, beforeEach, afterEach, it, describe } from "vitest";
 
 describe("My test", () => {
-  let recorder: Recorder;
+  // let recorder: Recorder;
 
-  beforeEach(async function (this: Context) {
-    recorder = await createRecorder(this);
+  beforeEach(async function () {
+    // recorder = await createRecorder(this);
```

Comment:
Does that mean the recorder is not working yet? if yes can we release this change?